index.bs

```diff
@@ -479,11 +544,12 @@ When the user agent wants to <dfn>run animators</dfn> in a given |workletGlobalS
 
   7. For each |timeline| in the <a>animator timeline list</a> of |definition|, do the following:
```

Comment:
Let's change |timeline| to |timelineOptions| so that it's less confusing to read.